### ISSUE TITLE
DAILY UPDATE AT DAY-END

### ISSUE THREAD

**Issue by dhruvin-restalgo** [RestAlgo/TradingApplication#200]
Enter Daily Updates with basic bullet points.

e.g.

  - Developed : user management in Admin login
  - Developing in progress : risk form design, limit management form etc.
  - Read book on sql
  - Took refence for designing from xyz place.

**Comment by Anil-Restalgo** [comment]
1) Deployment All Exe in Colo Server And InHouse Server.
2) Testing In Live Of Trading System With ViragBhai.

**Comment by Anil-Restalgo** [comment]
1. Testing In Live Of Trading System With ViragBhai.
2. Trade book orders need to be shown in Descending order.
3. TradeBook Filteration Change With LokeshSir.
4. TradeBook DataGridView Not Validating Issue With LokeshSir. (Pending)

**Comment by Anil-Restalgo** [comment]
1. Testing In Live Of Trading System With ViragBhai and VaibhavBhai And All Trading Team. 
2. API Download Whenever right Click on File And Click on Run Then Gif Image no Work Solved This Bug.
3. Send HeartBeat Logic Change With ViragBhai.

**Comment by Anil-Restalgo** [comment]
Present Date		:- 1) 15-06-2024
Task Given Date 	:- 1) 10-06-2024
					   2) 15-06-2024
					   3) 15-06-2024
Task Start Date		:- 1) 10-06-2024
					   2) 15-06-2024
					   3) 15-06-2024
Assigned to			:- Anil Nagrale
Assigned By 		:- ViragBhai
Module Name			:- API Download
Description			:- **1) Not Responding Issue in API Download.
					   2) Try to deactive rule on FNetLot & RNetLot columns is not working.
					   3) Sql Query Solution With AmanBhai.**
Current Status		:- 1) Hold
					   2) Done
					   3) Done
Estimated Date		:- 1) -
					   2) -
					   3) -
Merging Status		:- 1) -
					   2) -
					   3) -
Merging Date 		:- 1) -
					   2) -
					   3) -
Task End Date		:- 1) Hold
					   2) 15-06-2024
					   3) 15-06-2024
Tested By			:- 1) -
					   2) -
					   3) -
Tested Date			:- 1) -
					   2) -
					   3) -
Priority			:- 1) Normal 
					   2) Normal 
					   3) High

**Comment by Anil-Restalgo** [comment]
Present Date		:- 1) 17-06-2024
Task Given Date 	:- 1) 17-06-2024
					   2) 17-06-2024
					   3) 17-06-2024
					   4) 17-06-2024
Task Start Date		:- 1) 17-06-2024
					   2) 17-06-2024
					   3) 17-06-2024
					   4) 17-06-2024
Assigned to			:- Anil Nagrale
Assigned By 		:- ViragBhai
Module Name			:- 1) Trading System
Description			:- **1) Login Issue.
					   2) FNetLot 10 and FExecute Lot 12 show.
					   3) RuleWise Net Position Form Dockable Issue.
					   4) Live test on date 14-06-2024**
Current Status		:- 1) Done
					   2) Done
					   3) Done
					   4) Done
Estimated Date		:- 1) -
					   2) -
					   3) -
Merging Status		:- 1) -
					   2) -
					   3) -
Merging Date 		:- 1) -
					   2) -
					   3) -
Task End Date		:- 1) 17-06-2027
					   2) 17-06-2024
					   3) 17-06-2024
					   4) 17-06-2024
Tested By			:- 1) -
					   2) -
					   3) -
Tested Date			:- 1) -
					   2) -
					   3) -
Priority			:- 1) High 
					   2) High 
					   3) High
					   4) High

**Comment by Anil-Restalgo** [comment]
18-06-2024

========================================================
1). Deployment Of Updated Exe in Colo and In house PC.
2). Testing On Live With LokeshSir, PratikBhai and ViragBhai.
3). Rejection Order Issue Work with ViragBhai.
4). Mearge Code Of Issue Solved With ViragBhai.

**Comment by Anil-Restalgo** [comment]
19-06-2024
1). Deployment Of Updated Exe in Colo and In house PC.
2). Testing On Live With PratikBhai and ViragBhai.
3). Pending Order Issue Work with ViragBhai.
4). Issue Points Add in GitHub and Update With ViragBhai.

**Comment by Anil-Restalgo** [comment]
20-06-2024

1). Deployment Of Updated Exe in Colo and In house PC.
2). Testing On Live With PratikBhai and ViragBhai and Traders.
3). Helping Of Stuti.
4). APIDownload Target FileName Change File Copy Logic Call.

**Comment by Anil-Restalgo** [comment]
21-06-2024

1). Deployment Of Updated Exe in Colo and In house PC.
2). Testing On Live With PratikBhai and ViragBhai and Traders.
3). APIDownload Log Form Do Ready with ViragBhai.
4). APIDownload Dll Code Restructur With ViragBhai.

**Comment by Anil-Restalgo** [comment]
22-06-2024

1). Deployment Of Updated Exe in Support Team Testing Server.
2). Deployment of Updated Exe in Colo and inhouse PC of Nidhi Purohit's PC ,VibhavBhai's PC and General Pc and RakeshBhai's PC.
3). APIDownload Dll Code Restructur With ViragBhai.

**Comment by Anil-Restalgo** [comment]
24-06-2024

1). Testing On Live With PratikBhai and ViragBhai and Traders.
2). APIDownload Dll Code Restructur With ViragBhai.

**Comment by Anil-Restalgo** [comment]
25-06-2024

1). Testing On Live With PratikBhai and ViragBhai and Traders.
2). APIDownload Dll Code Restructur With ViragBhai.

**Comment by Anil-Restalgo** [comment]
26-06-2024

1). NEAT and NA Login with Viragbhai and IT Team.
2). APIDownload Dll Code Restructur With ViragBhai.
3). Do Recording Play in 0.36 PC For Testing With Viragbhai.

**Comment by Anil-Restalgo** [comment]
27-06-2024

1). Solve Error Of Sql Server Database 
2). APIDownload All Code Restructur With ViragBhai.

**Comment by Anil-Restalgo** [comment]
28-06-2024

1). APIDownload All Code Restructur With ViragBhai.

**Comment by Anil-Restalgo** [comment]
In FileTransplexer Project Update And Add Some Code Like Add CroneJob Class Add Line Of (1 to 40). SchedulerJob Class Add Line Of (1-53). In FrmMdiMain  Code Add  and Update Of Line(30-37,48,71-107,144-176,431-435).And In NSEFOLogin Class Update Line Of (20,85).
In HTTPDownload Update of Line (62,66).

**Comment by Anil-Restalgo** [comment]
In Transplexer project in RYDTTransplexerDownloderUtils Class In UnzipFile Method Change
Of Line No (220-320).in frmMDIMain_Final Update Line No (433).
Add Information Log in Entire Project Of RYDTTransplexer. 
Work In Colo With IT Team.

**Comment by Anil-Restalgo** [comment]
In Test Project Do Export to Excel and Learning On Async Await.
And Make a Demo and FrmAsyncAwait Add Line Of (1-129).

**Comment by Anil-Restalgo** [comment]
Learning On Async Await And Learning On MVC Task By Self.
Helping AmanBhai in TradingSystem - AdminModule in UserMaster Form.
Testing and developing In UserMaster with AmanBhai.

**Comment by Anil-Restalgo** [comment]
HardCode Remove From Gateway Simulation - Complete
Now Check Entire Gateway Simulation Logs And Add Logs - Continue

**Comment by Anil-Restalgo** [comment]
Now Check Entire Gateway Simulation Logs And Add Logs - Done
Mearge Code Of Sakshi in Trading System Developing Branch - Done
Load Contract File in Trading System Issue Of Sakshi Solved. - Done

**Comment by Anil-Restalgo** [comment]
**08-07-2024**

**Index View**
Percentage % column header proper.
This form should not be close on escape.
Dockable facility removed.
Right click to add token option.
Token search by indices name.

**Index Average**
Add token by right click.
New token add by F5

**Trading System**
If application by close button then ask message "Do you want to exit ?" and focus on "No" option.
In title bar show USERID instead of User Name.

**Done All Above Point.**

**Comment by Anil-Restalgo** [comment]
**09-07-2024**

**Index View**

Percentage % column header proper.
This form should not be close on escape.
Dockable facility removed.
Right click to add token option.
Token search by indices name.


**Index Average**

Add token by right click.
New token add by F5


**Trading System**

If application by close button then ask message "Do you want to exit ?" and focus on "No" option.
In title bar show USERID instead of User Name.

Testing Of This All Point with Viragbhai.
After This Indices form Notresponding issue Solved.
TradeBook Ascending Descending Order Invalidate issue in DataGridView Solved.

**Comment by Anil-Restalgo** [comment]
TradeBook Ascending Descending Order Invalidate issue Testing With LokeshSir And Invalidate Issue Solved With Lokesh Sir.
Give Transplexer Demo By Me to SupportTeam.
Update SQL Server Database In Live And Testing Of DataBase With RakeashBhai And AmanBhai.

**Comment by Anil-Restalgo** [comment]
New Exe Deployment in Gratias and Testing.
Helping Of Stuti on her code of server client.
Helping of AmanBhai in His SQL Query.
IOC order handel in Gateway Simulation.

**Comment by Anil-Restalgo** [comment]
**12-07-2024**

IOC order handel in Gateway Simulation.

**Comment by Anil-Restalgo** [comment]
**13-07-2024**

Testing Of New Algo Spinner With ViragBhai.
Learning on Asp.net Core.
Add TokenSubsribeManager In Trading System.

**Comment by Anil-Restalgo** [comment]
Testing Of New TradingSystem Exe With ViragBhai and Traders.
SQL Server Database Make Clone Copy For Testing.
ButterFly Strategy With ViragBhai.

**Comment by Anil-Restalgo** [comment]
**19-07-2024**

Testing Of New TradingSystem Exe DhruvinSir ,Client,VaibhavBhai and Trader.
Virtually grid and add last row for sum.
In frmMessage MessageBox Lines Solution.

**Comment by Anil-Restalgo** [comment]
**20-07-2024**
Virtually grid and add last row for sum.
Helping Sakshi on Her Code.

**Comment by Anil-Restalgo** [comment]
**24-07-2024**

Market depth(F5) should open on every window as it is itself a different window.
Live Exe Testing With ViragBhai and Trader.

**Comment by Anil-Restalgo** [comment]
**25-07-2024**

Market depth(F5) should open on every window as it is itself a different window.
In Live Exe Testing Of FS RS With ViragBhai,LokeshSir,VaibhavBhai and Trader.
Notification Message Box repair (re-structure).

**Comment by Anil-Restalgo** [comment]
**26-07-2024**

Market depth(F5) should open on every window as it is itself a different window Apply In All Forms.(Done)

**Comment by Anil-Restalgo** [comment]
**27-07-2024**

Notification Message Box repair.
Runnning Task Code Updated In Netposition.
Whenever Netposition Form Open At time CPU Consumption high Percentage this Issue Solved.

**Comment by Anil-Restalgo** [comment]
**29-07-2024**

Trend Color R&D.
Testing With LokeshSir & Trader In Live EXE.
Deploy App Exe in 36 Server Without Gateway Delegate EXE.
SpanFile Load Issue Work With PratikBhai.

**Comment by Anil-Restalgo** [comment]
**30-07-2024**

Testing With LokeshSir & Trader In Live EXE.
Deploy All New Exe in 36 Server and Gateway Delegate EXE Deploy in 30 in Live And After LokeshSir Discussion Revert Changes.
Give Demo Of FileDownload Transplexer of Support Team.

**Comment by Anil-Restalgo** [comment]
**31-07-2024**

Testing With ViragBhai & Trader In Live EXE.
Mearge Code With ViragBhai Of TradeBook New Code In Live Trading System.
After Code Mearge Testing New Exe With DeepakBhai.
RYDT Transplexer File Not Download Check In 36 Server.

**Comment by Anil-Restalgo** [comment]
**01-08-2024**

Testing With ViragBhai & Trader In Live EXE.
Index View and Index Average Changes Make a Demo and apply in Live.
Mearge Code With ViragBhai Index View Trend Column Add For Up Down Trend.
RYDT Transplexer File Not Download Check In 36 Server.
Merge Code Of Kartik Bhai Of Normal market watch - expire token replace with new token.
Finding Esc Key Issue With Usha Madam In AlgoMarketWatch NetLot Edit then Press Escp.

**Comment by Anil-Restalgo** [comment]
**02-08-2024**

Testing With ViragBhai & Trader In Live EXE.
Finding Issue With ViragBhai in Spinner.
Installation In NXBloack Client Server All Exe And API.


**03-08-2024**

Work With Support team for Mock.
Admin Module Not work in NXBloack Client Solved this Issue.
Sql Server DataBase Related Work With ITTeam and SupportTeam.
Installation In NXBloack Client Server Pc SqlDatabase With Support team.


**05-08-2024**

In house 36 Server Sql Server Related Issue Solved.
In NXBloack Client Server Pc Not Getting BroadCast Problem Solved.
Testing With ViragBhai & Trader In Live EXE.
Understanding new Work From LokeshSir.
Making Json File For New Developing.
Given Work On New Development Give Sakshi work For Designing and Crud with JSON.
Making Flow of New Development.

**Comment by Anil-Restalgo** [comment]
**06-08-2024**

Helping AmanBhai For His Code Of DataGridViewImageColumn.
Make a demo For new Development and R&D.

**Comment by Anil-Restalgo** [comment]
**08-08-2024**

Work With ViragBhai And Testing With ViragBhai.



**09-08-2024**

Testing With ViragBhai & SupportTeam In Live EXE.
Testing In Live With ViragBhai and SupportTeam.
Working with pratikBhai In UserMaster.

**Comment by Anil-Restalgo** [comment]
**10-08-2024**

AlgoMarketWatch Code Restructure.

**Comment by Anil-Restalgo** [comment]
**14-08-2024**

Support Team Netposition not Show Issue Has been Solved.
Find Issue Of RuleWiseNetPosition.

**Comment by Anil-Restalgo** [comment]
**16-08-2024 & 17-08-2024**

TaskID 411 Error in json for netposition and rulewise netposition and appglobal.
Finding Issue In RuleWiseNetPosition of GridView.

**Comment by Anil-Restalgo** [comment]
**20-08-2024**

Finding Issue In RuleWiseNetPosition of GridView.
Work With LokeshSir On EventView Module.

**Comment by Anil-Restalgo** [comment]
**21-08-2024**

Work With LokeshSir On EventView Module.

**Comment by Anil-Restalgo** [comment]
**22-08-2024  & 23-08-2024**

Work With LokeshSir On EventView Module.
Merge Code Of Error Solved Of json File Error Log.
Solved Json File Error Log With VMD.
UPM T410 Error Find With UPM and VMD.
Rule Wise Netposition Error Find and Write Log in this.

**Comment by Anil-Restalgo** [comment]
**24-08-2024**

TradeBook F8 issue while first time run form.
Work On EventView Work Given By LDS.

**Comment by Anil-Restalgo** [comment]
**28-08-2024 & 29-08-2024**

Work With LokeshSir On EventView.

**Comment by Anil-Restalgo** [comment]
**30-08-2024**

Work With LokeshSir On EventView.
Give Transplexer Demo of SupportTeam.

**Comment by Anil-Restalgo** [comment]
**31-08-2024**

Work On EventView.
Heping Of SupportTeam for installation Of Exe in Gratias Colo Server.

**01-09-2024 (Sunday)**

Work On EventView.

**Comment by Anil-Restalgo** [comment]
**02-09-2024     &&     03-09-2024**
======================

Work On EventView.

**Comment by Anil-Restalgo** [comment]
**04-09-2024**
===================

Work On EventView.
Put The Event View All Class And Forms TryCatch.
Whenever Set Property From PropertyWindow and Again Open Property Window For Set Property that time not Load Previous Property Issue.

**Comment by Anil-Restalgo** [comment]
**05-09-2024**
========

Work On Event View. 
Point Given By VaibhavBhai Like Start,Stop and Clear 
From Righ Click.(Done)(Testing Pending).
Give Demo Of Event View to Client.
Work With AbhishekBhai For StaticIP.

**Comment by Anil-Restalgo** [comment]
**06-09-2024**
=========

Work On Event View. 
Confirm of All Column From DhruvinSir.
Helping of Kartik bhai in SQlQuery.

**Comment by Anil-Restalgo** [comment]
**09-09-2024**
========

Work On Event View. 
Make DataTable and make Store Procedure For NSEFO.

**Comment by Anil-Restalgo** [comment]
**10-09-2024**
=====

Work With LokeshSir On Event View.

**Comment by Anil-Restalgo** [comment]
**11-09-2024**
=====

Work With LokeshSir On Event View. 
Give Demo Of Support Team Of Event Module.
Work With VMD And Support Team On Treding System Exe Of Delhi Client.

**Comment by Anil-Restalgo** [comment]
**12-09-2024**
==

Work With LokeshSir On Event View. 
Work With VMD And Support Team On Delegate Connection in Gratias Colo Server.

**Comment by Anil-Restalgo** [comment]
**13-09-2024**
=

Work On EventView GivenBy LokeshSir.

**Comment by Anil-Restalgo** [comment]
**14-09-2024**
=

Work On EventView GivenBy LokeshSir.

**Comment by Anil-Restalgo** [comment]
**15-09-2024**
=

Work On EventView GivenBy LokeshSir.

**Comment by Anil-Restalgo** [comment]
**17-09-2024**
=

Work On EventView GivenBy LokeshSir.

**Comment by Anil-Restalgo** [comment]
**18-09-2024**
=

Work On EventView GivenBy LokeshSir.
Work On New Task For Process Data For Make A file for EventView.
Testing Of Cm BroadCast With LokeshSir.

**Comment by Anil-Restalgo** [comment]
**19-09-2024**
=

Work On New Task For Process Data For Make A file for EventView.
Testing Of CM BroadCast With SupportTeam With StaticIP and local IP.
Check and Correction in Formula in JsonFile With SupportTeam.

**Comment by Anil-Restalgo** [comment]
**20-09-2024**
=

Work On New Task For Process Data For Make A file for EventView With LokeshSir.
Testing Of CM BroadCast With SupportTeam With StaticIP and local IP.

**Comment by Anil-Restalgo** [comment]
**23-09-2024**
=

In Event Module'S Any Changes At time Show in update not want to restart the application.
In Event module active deactive Run Time.
In Event module Create watch deprndency remove.
In Event module Combo name Change .

**Comment by Anil-Restalgo** [comment]
**24-09-2024**
=

Live Testing With Support team In Dhruvin Sir laptop and Other Laptop With Static IP Of Event View.
In Event Module change Whenever click on InBlipView, InMessageBar and AsDialogBox.

**Comment by Anil-Restalgo** [comment]
**25-09-2024**
=

Find The Link Of Files And All New Files add in Transplexer for download With Support Team.
In Transplexer Changes Like File Not Save on Destination Folder.
In Transplexer AlwaysSelected Changes Not work.

**Comment by Anil-Restalgo** [comment]
**26-09-2024**
=

Support Team ne Event Module na parameters and formulas samjavya.
In Transplexer Changes.

**Comment by Anil-Restalgo** [comment]
**27-09-2024**
=

Do Changes In Transplexer.
EventView All Script Save Logic Change For Multiple Script Add In Json File.

**Comment by Anil-Restalgo** [comment]
**28-09-2024**
=


In EventView Add New Logic for GetToken From All Scripts.

**Comment by Anil-Restalgo** [comment]
**30-09-2024 && 01-10-2024**
=

In EventView Add New Logic for GetToken From All Scripts.
NSECM BroadCast not Get From 36 server Client Issue With SupportTeam.

**Comment by Anil-Restalgo** [comment]
**02-10-2024**
=

In EventView Add New Logic for GetToken From All Scripts And DataGridView Selected Script Remove From ScriptList.

**Comment by Anil-Restalgo** [comment]
**03-10-2024**
=

NSECM BroadCast not Get From 36 server Client Issue With SupportTeam.
In EventView Add New Logic for GetToken From All Scripts And DataGridView Selected Script Remove From ScriptList.
F5 Token Add By Selected Expiry Date.

**Comment by Anil-Restalgo** [comment]
**04-10-2024**
=

New Exe of Event Module Testing On Live.
Do Changes In EventModule Parameters And Check All Code.

**Comment by Anil-Restalgo** [comment]
**05-10-2024**
=


New Exe of Event Module Testing On Live.
Do Changes In EventModule Parameters And Check All Code.
Mearge in Master Branch And For Testing Give to Support Team.

**Comment by Anil-Restalgo** [comment]
**07-10-2024**
=

In Pronto Scheduler Not Work Whenever Yesterday Holiday It's Issue Solved.
For Pronto Scheduler Installation in 36 Server Help Of SupportTeam.
New Issue Create In Property Grid Whenever not Isnclude of Any form in Json File Then not Show any one property.

**Comment by Anil-Restalgo** [comment]
**08-10-2024**
=

New Issue Create In Property Grid Whenever not Isnclude of Any form in Json File Then not Show any one property. 
Merge In Admin Branch With PPB.
Merge In RYDTControl With PPB.

**Comment by Anil-Restalgo** [comment]
**09-10-2024**
=

Market depth(F5) should open on every window as it is itself a different window.Test And Merge In Live Trading Branch.With VMD. 
Apply common properties in all forms Testing Done And Merge with VMD in Live Trading Branch.

**Comment by Anil-Restalgo** [comment]
**10-10-2024**
=

GridColor Error Solved.

**Comment by Anil-Restalgo** [comment]
**11-10-2024**
=

In Single Event Time Whenever Active then High price Cross 2nd time Then Show In Message.

**Comment by Anil-Restalgo** [comment]
**15-10-2024**
=

In Single Event Time Whenever Active then High price Cross 2nd time Then Show In Message.

**Comment by Anil-Restalgo** [comment]
**16-10-2024**
=

In Single Event Time Whenever Active then High price Cross 2nd time Then Show In Message.
In this Whenever Do Rule Active Deactive then Not Apply Then Finding Of This Issue.
Do Proper Testing For Active Deactive Rules.

**Comment by Anil-Restalgo** [comment]
**17-10-2024**
=

In Single Event Time Whenever Active then High price Cross 2nd time Then Show In Message.
TaskID 517 And 514 Merge In SID Branch(Event Tracker) and Do Proper Testing.
Work With DRA And UPM Of Indices Not show in NXBlock Branch.

**Comment by Anil-Restalgo** [comment]
**18-10-2024**
=

TaskID 518 Merge In SID Branch(Event Tracker) and Do Proper Testing.
TaskID 248 Merge In SID Branch(Event Tracker) Testing Pending.

**Comment by Anil-Restalgo** [comment]
**21-10-2024**
=

Check In SID Branch Try Catch and Comment On Each changes.

**Comment by Anil-Restalgo** [comment]
**22-10-2024**
=

TaskID 493 Merge In SID Branch(Event Tracker) Testing Pending.
Work With DRA On GatewayDelegate Of Error In Channel Creation.
Work With SKP on Error log For Finding Errors From Error Log.

**Comment by Anil-Restalgo** [comment]
**23-10-2024**
=

Proper Merging Of TaskID 517,518,514 And Proper Testing Done.
Make Saprate LongRunningTask Folder And Make A Sapate Class For LongRunningTask OrderBook Save.

**Comment by Anil-Restalgo** [comment]
**24-10-2024**
=

Make Saprate LongRunningTask Folder And Make A Sapate Class For LongRunningTask All Save.

**Comment by Anil-Restalgo** [comment]
**25-10-2024**
=

Make Saprate LongRunningTask Folder And Make A Sapate Class For LongRunningTask All Save OrderBook,TradeBook,OrderBookHistory,ApplicationTick,PingPong.

**Comment by Anil-Restalgo** [comment]
**26-10-2024**
=

Make a ScriptMaster,MarketPicture,BroadCastMessage and BestFive Class With ProtoContract and Testing It is Proper Work Or not.
Find The While Loop which Should have Taken For Saparate.

**Comment by Anil-Restalgo** [comment]
**28-10-2024**
=

DPR and LPPR Not show in Market Depth whenever Change Value From Token Selector.
Work With VMD in Engine Of Write Contract File From Original to Our Copy.

**Comment by Anil-Restalgo** [comment]
**29-10-2024**
=

Learn .net by Link Of https://www.w3schools.com/cs/cs_arrays_multi.php.

**Comment by Anil-Restalgo** [comment]
**30-10-2024**
=

Work With VMD.For merge Branch and Check IOC Strategy not add in Algomarket Watch.

**Comment by Anil-Restalgo** [comment]
**22-11-2024**
=

RuleWiseNetPosition Restructure and stuck Issue Solved.
Added New Columns as Per ViragBhai tell New Columns.

**Comment by Anil-Restalgo** [comment]
**23-11-2024**
=

Testing Of RuleWiseNetPosition Proper Testing.
Work with ViragBhai in Spinner(RYDT-AlgoSpinner).

**Comment by Anil-Restalgo** [comment]
**25-11-2024 && 26-11-2024**
=

Testing In Live With VMD Support Team And Client.
Work with ViragBhai in AlgoRuleTemplate and Testing of AlgoRuleTemplate.

**Comment by Anil-Restalgo** [comment]
**27-11-2024**
=

Changes In AlgoRule Of NoOfRule Issue.
Work with ViragBhai in AlgoRuleTemplate and Testing of AlgoRuleTemplate and M2M and RuleNetLot Column Add in AlgoMarketWatch.

**Comment by Anil-Restalgo** [comment]
**28-11-2024**
=

Work with ViragBhai on Disparity.

**Comment by Anil-Restalgo** [comment]
**29-11-2024**
=

Work with ViragBhai on Disparity.

**Comment by Anil-Restalgo** [comment]
**30-11-2024**
=

Work with ViragBhai on FATP And RATP.

**Comment by Anil-Restalgo** [comment]
**02-12-2024**
=

Work with ViragBhai on FATP And RATP and Testing.

**Comment by Anil-Restalgo** [comment]
**03-12-2024**
=

Work with ViragBhai on FATP And RATP and Testing.
Work on CycleWiseATP as per give me viragBhai.

**Comment by Anil-Restalgo** [comment]
**04-12-2024  &&  05-12-2024**
=

Work on CycleWiseATP as per give me viragBhai.

**Comment by Anil-Restalgo** [comment]
**07-12-2024**
=

Work on CycleWiseATP as per give me viragBhai.

**Comment by Anil-Restalgo** [comment]
**09-12-2024**
=

Work on CycleWiseATP as per give me viragBhai.

**Comment by Anil-Restalgo** [comment]
**10-12-2024**
=

Work on CycleWiseATP as per give me viragBhai.

**Comment by Anil-Restalgo** [comment]
**11-12-2024**
=

Work on CycleWiseATP as per give me viragBhai.

**Comment by Anil-Restalgo** [comment]
**12-12-2024**
=

Work on CycleWiseATP as per give me viragBhai(Below).
1. Report Data Save
   => bin format
2. Report Data Load (CurrentDayFile as per order book tradebook)
   => bin format
3. Rule Clean rule 
   => RuleNo + _X
4. Rule Delete
   => No Change
5. Every Day new file create for save and load
6. If cycle complete then ActualFSRS show other wise 0.00
7. TokenNumber with ExecutedQty show like 35001 (15)
10. All added in RuleNumber filter.

**Comment by Anil-Restalgo** [comment]
**13-12-2024**
=

Work on CycleWiseATP as per give me viragBhai.
Check Proper 
Proper Log (information , fatal , debug)
try catch
and Testing.

**Comment by Anil-Restalgo** [comment]
**14-12-2024**
=

Merge CycleWiseATP In Master Branch Of ViragBhai.

**Comment by Anil-Restalgo** [comment]
**17-12-2024**
=

In CycleWiseATP Changes in FSRS Logic.

**Comment by Anil-Restalgo** [comment]
**18-12-2024**
=

In CycleWiseATP Changes in FSRS Logic.

**Comment by Anil-Restalgo** [comment]
**19-12-2024**
=

In CycleWiseATP Changes in FSRS Logic.
Merge in Live CycleWiseATP.

**Comment by Anil-Restalgo** [comment]
**20-12-2024**
=

In CycleWiseATP FSRS Logic Done In ConRev and Box and Get current AskingFSRS Price in CycleWiseATP and Add StrategyName Column.
Merge in Live CycleWiseATP.

**Comment by Anil-Restalgo** [comment]
**21-12-2024**
=

Work With ViragBhai.

**Comment by Anil-Restalgo** [comment]
**23-12-2024**
=

Work With ViragBhai in CycleWiseATP Report on Token NameChange and Execute Qty Related Changes.

**Comment by Anil-Restalgo** [comment]
**24-12-2024 && 26-12-2024**
=

Work With ViragBhai.
Work On TaskID 592 Of SpotPrice UpperSpotPrice and LowerSpotPrice.
Work On New Report Of StrategyLogReport Open From CycleWiseATPReport.

**Comment by Anil-Restalgo** [comment]
**27-12-2024**
=

Work On TaskID 593 of New Report Of StrategyLogReport Open From CycleWiseATPReport.

**Comment by Anil-Restalgo** [comment]
**30-12-2024**
=

Work On EventModule Issue.

**Comment by Anil-Restalgo** [comment]
**31-12-2024**
=

Work On EventModule Issue.
Add IndexView in SID Branch.

**Comment by Anil-Restalgo** [comment]
**01-01-2025**
=

Work On EpiryDate rollover in Event Module.

**Comment by Anil-Restalgo** [comment]
**02-01-2025**
=

Work On EpiryDate rollover in Event Module Done And Testing Complete my Side Merge Pending.

**Comment by Anil-Restalgo** [comment]
**03-01-2025**
=

Check TradingSystem - SID ErrorLog and Solve the errors.(Pending)
Make Test Database Server and make 15 test ID.

**Comment by Anil-Restalgo** [comment]
**04-01-2025 && 07-01-2025 && 08-01-2025**
=

Check TradingSystem - SID ErrorLog and Solve the errors.(Pending)
Work With VMD.

**Comment by Anil-Restalgo** [comment]
**09-01-2025**
=

Work With VMD on SID Branch.
Work With VMD on CycleWiseATP and CycleWiseATPDetailReport and Merge In LIVE.

**Comment by Anil-Restalgo** [comment]
**10-01-2025**
=

Work on TaskID 104 Trnasplexer Issue.
Work With VMD and Lokeshsir On Broadcast testing.

**Comment by Anil-Restalgo** [comment]
**11-01-2025**
=

Work on TaskID 104 Trnasplexer Issue.

**Comment by Anil-Restalgo** [comment]
**21-01-2025**
=

Work on TaskID 104 Trnasplexer Issue.

**Comment by Anil-Restalgo** [comment]
**22-01-2025**
=

Work on TaskID 104 Trnasplexer Issue.
And Work On FileDownload Path.

**Comment by Anil-Restalgo** [comment]
**23-01-2025**
=

Work on TaskID 104 Trnasplexer Issue.
And Work On FileDownload Path.

**Comment by Anil-Restalgo** [comment]
**24-01-2025**
=

Work on FileDownload Get Much More Time Issue(R&D).
Testing On Live With StaticIP With PratikBhai Chuahan.

**Comment by Anil-Restalgo** [comment]
**25-01-2025**
=

Testing With ViragBhai 10 ID In Colo Server For performance.

**Comment by Anil-Restalgo** [comment]
**27-01-2025**
=

Work On CycleWise ATP Ration Issue.
Testing With IT and Support team Of Static ID.

**Comment by Anil-Restalgo** [comment]
**28-01-2025**
=

Work On CycleWise ATP Ration Issue with VMD.

**Comment by Anil-Restalgo** [comment]
**29-01-2025**
=

Work On CycleWise ATP Ratio Issue with VMD.

**Comment by Anil-Restalgo** [comment]
**30-01-2025**
=

Work On CycleWise ATP Ratio Issue with VMD(SOLVED).
Testing and Merging In Live CycleWiseATP Report.
Working On 0 ratio actualFSRS get 0 value Solved With VMD.

**Comment by Anil-Restalgo** [comment]
**31-01-2025**
=

Work On CycleWise ATP Whenever Ratio Change Then Not Get last 2 Cycle actualFSRS.
Work On EventView module for ExpiryRollover.

**Comment by Anil-Restalgo** [comment]
**01-02-2025 && 03-02-2025 && 04-02-2025**
=

Work On EventView module for ExpiryRollover.
Add New Link in Transplexer For Diffrent type.
Work With VMD On Client Problem Of Auto Squre Off Testing.

**Comment by Anil-Restalgo** [comment]
**05-02-2025**
=

Add New Link in Transplexer For Diffrent type.
Testing Of StaticIP File Download With IT team and Support Team.

**Comment by Anil-Restalgo** [comment]
**07-02-2025**
=

Test in Live Multiple ID Active SupportTeam and VMD and PPB.
Changes in TradingSystem NetPosition Mearge With Dealer.
Testing BroadCast in Share India Server With SupportTeam and VMD.

**Comment by Anil-Restalgo** [comment]
**08-02-2025**
=

Testing BroadCast in Share India Server With SupportTeam and VMD.
Work On TaskID 620.

**Comment by Anil-Restalgo** [comment]
**10-02-2025**
=

Testing BroadCast in Share India Server With SupportTeam and VMD.
Work On TaskID 620 and Mearge in Live.
Static IP Testing File Download with VMD.

**Comment by Anil-Restalgo** [comment]
**10-02-2025**
=

Testing BroadCast in Share India Server With SupportTeam and VMD.
Work On TaskID 620 and Mearge in Live.
Work With VMD on TaskID 622.
Static IP Testing File Download with VMD.

**Comment by Anil-Restalgo** [comment]
**11-02-2025**
=

Testing Live EXE With SupportTeam and PPB.
Work With VMD on AlgoSpinnerIssue.
Installation New Updates In Live Colo With Support Team and Testing It ProperWork.

**Comment by Anil-Restalgo** [comment]
**12-02-2025**
=

Testing Live EXE With SupportTeam and PPB.
Do Transplexer Changes. 
Create All ID With Support Team For Simulation Perform.

**Comment by Anil-Restalgo** [comment]
**13-02-2025**
=

Changes In ProntoScheduler Add Set Path For File Distination.
Do Transplexer Changes. 
Create All ID With Support Team For Simulation Perform.

**Comment by Anil-Restalgo** [comment]
**21-02-2025**
=

FileDownload Issue On Trading Login.

**Comment by Anil-Restalgo** [comment]
**22-02-2025**
=

FileDownload Issue On Trading Login Testing.
Work With VMD on Delegate.
Check All Live EXE Log Folders and Database With Support Team.

**Comment by Anil-Restalgo** [comment]
**24-02-2025**
=

Check IIS Problem On ShareIndia Server With Support Team.
Work With VMD on Delegate.

**Comment by Anil-Restalgo** [comment]
**25-02-2025**
=

Work With VMD on Delegate.

**Comment by Anil-Restalgo** [comment]
**27-02-2025**
=

Work With VMD on Delegate.

**Comment by Anil-Restalgo** [comment]
**28-02-2025**
=

Work on Live Exe Start and put all files and License and new ShareIndia Exe Testing With Support Team.
Work With VMD on Delegate.
Work On Algo market Depth Changes As per VaibhavBhai Discussion.

**Comment by Anil-Restalgo** [comment]
**01-03-2025**
=

Work On Algo market Depth Changes As per VaibhavBhai Discussion.

**Comment by Anil-Restalgo** [comment]
**03-03-2025**
=

Changes In FileDownloader In SID Branch.
Work On market Depth Changes As per VaibhavBhai Discussion.

**Comment by Anil-Restalgo** [comment]
**04-03-2025**
=

Work On market Depth Changes As per VaibhavBhai Discussion and Mearge In Master Branch

**Comment by Anil-Restalgo** [comment]
**05-03-2025**
=

Work On Algo Rule ConRev Rule Bug.

**Comment by Anil-Restalgo** [comment]
**06-03-2025**
=

Work On Algo Rule ConRev Rule Bug. and 
1. CR 
2. Butterfly
3. Box
4. PriceDIff_IOC
5. CR_IOC
6. Box_IOC
              In All This Strategy Not Show Delta Now Showing.
7. AlgoRule Utariye Tyare ConRev Strategy ni ander Second Leg Noto aavto a thy Gayu Che.
8. AlgoRule Utariye tyare je 1st rule ma je strike price hoy a Second leg ma noti aavti a thy gayu che.
9. MarketDepth ni ander jyare Exchange pr enter karta hata to je strike price select kari hoy a noti aavti biji strike price aavi jati hati a thy gayu che.

**Comment by Anil-Restalgo** [comment]
**07-03-2025**
=

MarketWatch Blink Color Kase Pan click karu to work karvu joiye and agar a form pachad jay to color blink bankd thy javu joiye.

**Comment by Anil-Restalgo** [comment]
**08-03-2025**
=

New Server Installation With It Team and Support Team and Testing Proper Working Or not.

**Comment by Anil-Restalgo** [comment]
**11-03-2025**
=

WorkOn TaskID 673 and TaskID 644 and Work With VMD.

**Comment by Anil-Restalgo** [comment]
**13-03-2025**
=

WorkOn TaskID 682.

**Comment by Anil-Restalgo** [comment]
**18-03-2025 && 19-03-2025**
=

WorkOn TaskID 682.

**Comment by Anil-Restalgo** [comment]
**20-03-2025**
=

WorkOn TaskID 682 and TaskID 632.

**Comment by Anil-Restalgo** [comment]
**21-03-2025**
=

WorkOn TaskID 682 and TaskID 632.

**Comment by Anil-Restalgo** [comment]
**22-03-2025**
=

WorkOn TaskID 682.

**Comment by Anil-Restalgo** [comment]
**24-03-2025**
=

WorkOn TaskID 682 Do Changes As Per Discuss With VMD.

**Comment by Anil-Restalgo** [comment]
**25-03-2025**
=

Work With VMD on Gateway Adapter and CTCLID.

**Comment by Anil-Restalgo** [comment]
**26-03-2025**
=

WorkOn TaskID 650.

**Comment by Anil-Restalgo** [comment]
**27-03-2025**
=

WorkOn TaskID 650 Done.

**Comment by Anil-Restalgo** [comment]
**28-03-2025**
=

Work With SKP on TaskID 672.

**Comment by Anil-Restalgo** [comment]
**29-03-2025**
=

Work With VMD.

**Comment by Anil-Restalgo** [comment]
**31-03-2025**
=

Work On Transplexer For Ael File downloading Issue.
TaskID650 Mearge in SupportTesting Branch With VMD.

**Comment by Anil-Restalgo** [comment]
**01-04-2025**
=

Work With VMD On New Development of Lot Square Off.

**Comment by Anil-Restalgo** [comment]
**08-04-2025**
=

Work On TaskID 660.

**Comment by Anil-Restalgo** [comment]
**09-04-2025**
=

Work On TaskID 650.

**Comment by Anil-Restalgo** [comment]
**10-04-2025**
=

Work On TaskID 650.

**Comment by Anil-Restalgo** [comment]
**25-04-2025**
=

Work On TaskID 660 with VMD.

**Comment by Anil-Restalgo** [comment]
**03-05-2025 && 05-05-2025**
=

Work On TaskID 752 And TaskID 772.

**Comment by Anil-Restalgo** [comment]
**06-05-2025**
=

Work On TaskID 772 And 1 Path Write Multiple Times change to 1 common method to all with VMD and Mearge.

**Comment by Anil-Restalgo** [comment]
**07-05-2025**
=

Changes in Admin As It is Trading System Of TaskID 682 , 745, 772 and mearge in UAT with Aman Bhai.

**Comment by Anil-Restalgo** [comment]
**08-05-2025**
=

Work On TaskID 744.

**Comment by Anil-Restalgo** [comment]
**09-05-2025 && 10-05-2025**
=

Work On TaskID 744 Add in Admin Module.
Work With VMD of BSEFO.

**Comment by Anil-Restalgo** [comment]
**13-05-2025**
=

Work On TaskID 745 Whenever Logout Application And Login Again at time broadcast not Work.
Mearge Task Of 682 and 745 In Admin Module.

**Comment by Anil-Restalgo** [comment]
**14-05-2025**
=

Work On SID Condition in SID Branch.
Work On TaskID 745 For NetPosition Grid InValidate.

**Comment by Anil-Restalgo** [comment]
**15-05-2025**
=

Work On TaskID 745 For NetPosition Grid InValidate and After Logout New User Login Enable Do.

**Comment by Anil-Restalgo** [comment]
**16-05-2025**
=

Work On RYDTEngine With VMD And PPB Of Key Not Fing In Dicitionary.

**Comment by Anil-Restalgo** [comment]
**17-05-2025**
=

Meating With PratikBhai,ViragBhai and LokeshSir On RYDTEngine.
Work On TaskID 745 For NetPosition Grid InValidate and After Logout New User Login Enable Do.

**Comment by Anil-Restalgo** [comment]
**20-05-2025**
=

Work On BSEFO Testing Of TaskID 803.

**Comment by Anil-Restalgo** [comment]
**21-05-2025**
=

Work On BSEFO Testing and Changes In "NSEFO" Harcode of TaskID 803 and Merge For Testing.
WOrk With VMD On NSEFO-Delegate.

**Comment by Anil-Restalgo** [comment]
**22-05-2025**
=

Work On TaskID 806 In GatwaySimulator And Work On TaskID 638 in RYDT Ops Engine and Work On TaskID 809 in TradingSystem.

**Comment by Anil-Restalgo** [comment]
**23-05-2025**
=

Work On Transplexer.

**Comment by Anil-Restalgo** [comment]
**24-05-2025**
=

Work On Transplexer.
Work On TaskID 814.

**Comment by Anil-Restalgo** [comment]
**26-05-2025**
=

Mearge Task OF 743,728,770,773,774,800 in RYDT Engine Master Branch.

**Comment by Anil-Restalgo** [comment]
**27-05-2025**
=

1. Admin Module 
	TaskID 728,758, 660, 774, 744, 682, 731, 800, 811
2. RYDT Ops Engine
	Task OF 743,728,770,773,774,800 Testing In Live.
This All Task Testing In Live.

**Comment by Anil-Restalgo** [comment]
**28-05-2025**
=

1. RYDT Log Manager
	Work With AmanBhai Of OrderBook,Tradebook Save in Databse.
2. RYDT Ops Engine
	Task OF 810, 759 Merge in Testing Branch and TaskID 810 Merge in Develope and Live Branch.

**Comment by Anil-Restalgo** [comment]
**29-05-2025**
=

1. TradingSystem_AdminModule
	Work With AmanBhai Of NetQty,OSQty not Update Properly After Logout.
2. KT
	Meating With PratikBhai and LokeshSir For KT.

**Comment by Anil-Restalgo** [comment]
**30-05-2025**
=

1. TradingSystem
	Logout Issue.

**Comment by Anil-Restalgo** [comment]
**06-06-2025**
=

1. GatewaySimulator
	TaskID 806.
	76 na badle 69 number nu version revert kariu che. becasue 76 version ma kai j samjatu notu and pratikbhai ane sir jode 
	vat kari pn emne pn kai khayal noto so 69 number version is revert in NSEFO and BSEFO.

**Comment by Anil-Restalgo** [comment]
**07-06-2025**
=

1. RYDTEngine
	TaskID 823.
	LookUp Risk.

**Comment by Anil-Restalgo** [comment]
**09-06-2025**
=

1. TradingSystem - SID
	Not get BroadCast Error show in ScriptMasterPublish Solved.

2. RYDT Engine
	LookUp Risk.

**Comment by Anil-Restalgo** [comment]
**10-06-2025**
=

1. RYDT Engine
	LookUp Risk.

**Comment by Anil-Restalgo** [comment]
**12-06-2025**
=

1. RYDT Transplexer
	TaskID 807 BSEFO Fils Related.

**Comment by Anil-Restalgo** [comment]
**13-06-2025**
=

1. RYDT Engine
	TaskID 833 Fix NSEFO check and how to solve for BSEFO start.

2. TradingSystem
	TaskID 833 Fix NSEFO check and how to solve for BSEFO start.
	
3. TradingSystem - AdminModule
	TaskID 833 Fix NSEFO check and how to solve for BSEFO start.

**Comment by Anil-Restalgo** [comment]
**16-06-2025**
=

1. RYDT Engine
	TaskID 836 BSEFO and NSEFO limit report issue in carry position..

2. TradingSystem - AdminModule
	TaskID 836 BSEFO and NSEFO limit report issue in carry position..

**Comment by Anil-Restalgo** [comment]
**17-06-2025**
=

1. RYDT Engine
	TaskID 836 BSEFO and NSEFO limit report issue in carry position..

2. TradingSystem - AdminModule
	TaskID 836 BSEFO and NSEFO limit report issue in carry position..

**Comment by Anil-Restalgo** [comment]
**18-06-2025**
=

1. RYDT Engine
	TaskID 837 NSEFO and BSEFO both segment bhavcopy read in engine and close price given to LTP in Opening NetPosition.

**Comment by Anil-Restalgo** [comment]
**20-06-2025**
=

1. RYDT Engine
	TaskID 837 NSEFO and BSEFO both segment bhavcopy read in engine and close price given to LTP in Opening NetPosition.

**Comment by Anil-Restalgo** [comment]
**21-06-2025**
=

1. RYDT Engine
	TaskID 837 NSEFO and BSEFO both segment bhavcopy read in engine and close price given to LTP in Opening NetPosition.

**Comment by Anil-Restalgo** [comment]
**23-06-2025 to 25-06-2025**
=

1. RYDT Engine
	Margin rejection...

**Comment by Anil-Restalgo** [comment]
**26-06-2025**
=

1. RYDT Engine
	**TaskID 851** 
	Dealer Margin is 100000 but Wenever Dealer Do Order and Dealer not reach margin value it's before get rejection of Margin. 
	RYDTRMSSummaryUpdate not response properly to admin and client.
	
2. TradingSystem 
	**TaskID 830** 
	RYDTRMSSummaryUpdate Not Get response properly .
	
3. TradingSystem - AdminModule
	**TaskID 830**
	RYDTRMSSummaryUpdate Not Get response properly.

**Comment by Anil-Restalgo** [comment]
**27-06-2025**
=

1. RYDT Engine
	TaskID 830
	Fund Form.
	
2. TradingSystem 
	TaskID 830 
	Fund Form.
	
3. TradingSystem - AdminModule
	TaskID 830
	Fund Form.

**Comment by Anil-Restalgo** [comment]
**30-06-2025**
=

1. RYDT Engine
	TaskID 830
	Fund Form.
	
2. TradingSystem 
	TaskID 830 
	Fund Form.
	
3. TradingSystem - AdminModule
	TaskID 830
	Fund Form.
	
3. Transplexr - AdminModule
	TaskID 807
	Fund Form.

**Comment by Anil-Restalgo** [comment]
**30-06-2025**
=

1. RYDT Engine
	Margin 0 show while order reject or running ordering.
	and Margin Stuck in Square off All.

**Comment by Anil-Restalgo** [comment]
**01-07-2025 to 03-07-2025**
=

1. RYDT Engine
	Margin 0 show while order reject or running ordering.
	and Margin Stuck in Square off All.
	Single strike SQ ma margin issue.**(TaskID 858)**
	In Opening NP, RISK net position and limit report in sub branch and main HO not getting proper.**(TaskID 859)**

**Comment by Anil-Restalgo** [comment]
**04-07-2025**
=

1. RYDT Engine
	In Opening NP, RISK net position and limit report in sub branch and main HO not getting proper.(TaskID 859)
	Single strike SQ ma margin issue.(TaskID 858)

**Comment by Anil-Restalgo** [comment]
**05-07-2025**
=

1. RYDT Engine
	Single strike SQ ma margin issue Check Flow and Check Risk Check or not.(TaskID 858)
	Check Risk In Normal and algo orders.

**Comment by Anil-Restalgo** [comment]
**07-07-2025 && 08-07-2025**
=

1. RYDT Engine
	Single strike SQ ma margin issue Check Flow and Check Risk Check or not.(TaskID 858)
	Check Risk In Normal and algo orders.

**Comment by Anil-Restalgo** [comment]
**14-07-2025**
=

1. RYDT Engine
	TaskID 869.

**Comment by Anil-Restalgo** [comment]
**15-07-2025**
=

1. RYDT Engine
	TaskID 869.

**Comment by Anil-Restalgo** [comment]
**16-07-2025**
=

1. RYDT Engine
   If Order 2 diffrent dealer orders then admin margin get not 0 in Carry.

**Comment by Anil-Restalgo** [comment]
**22-07-2025**
=

1. RYDT Engine
   If Order 2 diffrent dealer orders then admin margin get not 0 in Carry.

**Comment by Anil-Restalgo** [comment]
**23-07-2025**
=

1. RYDT Engine
   If Order 2 diffrent dealer orders then admin margin get not 0 in Carry.

**Comment by Anil-Restalgo** [comment]
**24-07-2025**
=

1. RYDT Engine
   If Order 2 diffrent dealer orders then admin margin get not 0 in Carry.

**Comment by Anil-Restalgo** [comment]
**25-07-2025**
=

1. RYDT Engine
   If Order 2 diffrent dealer orders then admin margin get not 0 in Carry.

**Comment by Anil-Restalgo** [comment]
**26-07-2025 && 28-07-2025**
=

1. RYDT Engine
   If Order 2 diffrent dealer orders then admin margin get not 0 in Carry With LokeshSir.
2. Gateway
   Partially Qty Not get Order Confirmation With VMD.

**Comment by Anil-Restalgo** [comment]
**29-07-2025**
=

1. RYDT Engine
   BSEFO Margin Issue.

**Comment by Anil-Restalgo** [comment]
**30-07-2025**
=

1. RYDT Engine
   BSEFO Margin Issue.

**Comment by Anil-Restalgo** [comment]
**02-08-2025**
=

1. RYDT Engine & TradingSystem
   Client level P&L Set TaskID 886.

**Comment by Anil-Restalgo** [comment]
**04-08-2025**
=

1. RYDT Engine & TradingSystem
   Client level P&L Set TaskID 886.

**Comment by Anil-Restalgo** [comment]
**05-08-2025**
=

1. RYDT Engine & TradingSystem
   Client level P&L Set TaskID 886 and TaskId 869.

**Comment by Anil-Restalgo** [comment]
**06-08-2025**
=

1. RYDT Engine & TradingSystem
   Client level P&L Set TaskID 886 Testing With DipakBHai and TaskId 869 Changes as Pr discussed With Dhruvin Sir.

**Comment by Anil-Restalgo** [comment]
**07-08-2025**
=

1. RYDT Engine & Delegate
   Retransmit Trade Work with ViragBhai.

**Comment by Anil-Restalgo** [comment]
**08-08-2025**
=

1. RYDT Engine & Delegate
   Retransmit Trade Work with ViragBhai.

**Comment by Anil-Restalgo** [comment]
**12-08-2025**
=

1. Trading System
   TaskID 886 Changes as per disuss with DipakBhai.

**Comment by Anil-Restalgo** [comment]
**13-08-2025**
=

1. Trading System
   Margin Refresh Issue Testing With Support Team.

**Comment by Anil-Restalgo** [comment]
**14-08-2025**
=

1. RYDT Engine.
   TaskID 869 Done LockObj put in Engine For RiskNetPosition Not Update Proper.

**Comment by Anil-Restalgo** [comment]
**18-08-2025**
=

1. RYDT Engine.
   TaskID 895.

**Comment by Anil-Restalgo** [comment]
**19-08-2025**
=

1. RYDT Engine.
   TaskID 895.

**Comment by Anil-Restalgo** [comment]
**20-08-2025**
=

1. RYDT Engine.
   TaskID 895.

**Comment by Anil-Restalgo** [comment]
**21-08-2025**
=

1. RYDT Engine.
   TaskID 895.

**Comment by Anil-Restalgo** [comment]
**22-08-2025**
=

1. RYDT Engine.
   TaskID 896.

**Comment by Anil-Restalgo** [comment]
**25-08-2025**
=

1. RYDT Engine and Trading System.
   TaskID 896.

**Comment by Anil-Restalgo** [comment]
**26-08-2025**
=

1. RYDT Engine and Trading System.
   TaskID 896 Mearge in UAT.

**Comment by Anil-Restalgo** [comment]
**28-08-2025**
=

1. RYDT Engine and Trading System.
   TaskID 854 Margin 0 show while order reject or running ordering This Work But not Get 0 in BhavNagar Server.

**Comment by Anil-Restalgo** [comment]
**29-08-2025**
=

1. RYDT Engine.
   TaskID 586 and Spinner Not Connected With 2 Trading in Live Issue.

**Comment by Anil-Restalgo** [comment]
**30-08-2025**
=

1. RYDT Engine.
   TaskID 586 And TaskID 915.

**Comment by Anil-Restalgo** [comment]
**01-09-2025**
=

1. RYDT Engine.
   Mearge in Live All Pending Point with VMD.
2. TradingSystem
   Put All Check Points pdf in All TaskID GitHub.

**Comment by Anil-Restalgo** [comment]
**02-09-2025**
=

1. RYDT Engine.
   TaskID 915 Done.

**Comment by Anil-Restalgo** [comment]
**05-09-2025**
=

1. RYDT Engine.
   Margin Get 0 in ToolStrip Menu Whenever Get Margin Rejection.

**Comment by Anil-Restalgo** [comment]
**06-09-2025**
=

1. RYDT Engine.
   Margin Get 0 in ToolStrip Menu Whenever Get Margin Rejection.

**Comment by Anil-Restalgo** [comment]
**08-09-2025**
=

1. RYDT Engine.
   Margin Get 0 in ToolStrip Menu Whenever Get Margin Rejection.

**Comment by Anil-Restalgo** [comment]
**09-09-2025**
=

1. RYDT Engine.
   Margin Get 0 in ToolStrip Menu Whenever Get Margin Rejection With LokeshSir.
   TaskID 586.

**Comment by Anil-Restalgo** [comment]
**10-09-2025**
=

1. RYDT Engine.
   TaskID 586.
   TaskID 648 R&D Try To Solve.

**Comment by Anil-Restalgo** [comment]
**11-09-2025**
=

1. RYDT Engine.
   Margin Get 0 in ToolStrip Menu Whenever Get Margin Rejection With LokeshSir.

**Comment by Anil-Restalgo** [comment]
**12-09-2025**
=

1. RYDT Engine.
   Margin Get 0 in ToolStrip Menu Whenever Get Margin Rejection With LokeshSir.
2. TradingSystem
   TaskID 930.

**Comment by Anil-Restalgo** [comment]
**13-09-2025**
=

1. RYDT Engine.
   => On Square Off All After get Last Trade At Time Get Sq Off Process Finish Message immediate.
   => <RecalculateM2MForAllUsers>d__6 - Bw9LopIcmsqKpg37RC1R - System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
      at System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
      at System.Collections.Generic.Dictionary`2.ValueCollection.Enumerator.MoveNext()
      at RYDTServer.RISK.RefreshRiskAllParams.<RecalculateM2MForAllUsers>d__6.MoveNext() in D:\Live\VMD\RYDTEngine_NEW\RYDTServer\RISK\RefreshRiskAllParams.cs:line 189
	Solved This Error Testing Pending.
2. TradingSystem
   TaskID 930 Changes Like blank accepted decimal value get in 1.5K,1.5L,1.5C In This method.

**Comment by Anil-Restalgo** [comment]
**15-09-2025**
=

1. RYDT Engine.
   TaskID 932.
2. TradingSystem - AdminModule
   TaskID 854.

**Comment by Anil-Restalgo** [comment]
**16-09-2025**
=

1. RYDT Engine.
   Changes As Per Some performance level and test not make any TaskID and Branch.

**Comment by Anil-Restalgo** [comment]
**17-09-2025**
=

1. In Local.
   Try To Solved TaskID 934.

**Comment by Anil-Restalgo** [comment]
**18-09-2025**
=

1. TradingSystem.
   TaskID 936.

**Comment by Anil-Restalgo** [comment]
**19-09-2025**
=

1. TradingSystem.
   TaskID 936  and TaskID 942.

**Comment by Anil-Restalgo** [comment]
**20-09-2025**
=

1. RYDT Engine.
   TaskID 942.

**Comment by Anil-Restalgo** [comment]
**22-09-2025**
=

1. RYDT Engine.
   TaskID 942 and Testing Of AutoSqOff Message not Get After Trade And margin uper chadi Jay che.

**Comment by Anil-Restalgo** [comment]
**23-09-2025**
=

1. RYDT Engine.
   TaskID 942 Mearge in UAT With VMD And Some Changes By VMD Beacuse Some Place Code not Write Properly.

**Comment by Anil-Restalgo** [comment]
**24-09-2025**
=

1. RYDT Engine.
   Margin Uper thy jatu hatu and Rejection aavi jay che a test.

**Comment by Anil-Restalgo** [comment]
**25-09-2025**
=

1. RYDT Engine.
   TaskID 944.

**Comment by Anil-Restalgo** [comment]
**26-09-2025**
=

1. RYDT Engine.
   TaskID 944 Margin Uper thy jatu hatu and Rejection aavi jay che a test.

**Comment by Anil-Restalgo** [comment]
**27-09-2025**
=

1. RYDT Engine.
   TaskID 944 Margin Uper thy jatu hatu and Rejection aavi jay che a test.

**Comment by Anil-Restalgo** [comment]
**30-09-2025**
=

1. RYDT Engine.
   TaskID 944.
   AlgoMarketWatch Data get slow Watch With Lokesh Sir.
   SingleStrike SqOff Not Work With Support Testing.

**Comment by Anil-Restalgo** [comment]
**01-10-2025**
=

1. Trading System.
   TaskID 951.
   AlgoMarketWatch Data get slow SupportTeam.
   In Live Get Extra Qty Check.
   In Live BroadCast Not Get Proper Check With Lokesh Sir and SupportTeam.

**Comment by Anil-Restalgo** [comment]
**03-10-2025**
=

1. Trading System.
   TaskID 951.
   In Live BroadCast Not Get Proper Check With Lokesh Sir and SupportTeam.

**Comment by Anil-Restalgo** [comment]
**04-10-2025**
=

1. Trading System.
   TaskID 951.

**Comment by Anil-Restalgo** [comment]
**07-10-2025**
=

1. RYDTEngine.
   NSECM Risk Meating With Lokesh Sir and ViragBhai and VaibhavBhai.
   Learning NSECM Risk From watch to NSEFO Risk Step by step how to do(Continue).

**Comment by Anil-Restalgo** [comment]
**08-10-2025**
=

1. RYDTEngine.
   NSECM Risk Meating With Lokesh Sir and ViragBhai and VaibhavBhai.
   Get Wrong Margin Rjection In Live .
   Learning NSECM Risk From watch to NSEFO Risk Step by step how to do(Continue).

**Comment by Anil-Restalgo** [comment]
**09-10-2025**
=

1. TradingSystem.
   Testing With Support Team Of Wrong Margin Rjection In Live.

**Comment by Anil-Restalgo** [comment]
**10-10-2025**
=

1. TradingSystem.
   Look Of Wrong Margin Rjection In Live.
2. RYDTEngine
   Learning NSECM Risk From watch to NSEFO Risk Step by step how to do(Continue).

**Comment by Anil-Restalgo** [comment]
**11-10-2025**
=

1. RYDTEngine.
   Look Of Wrong Margin Rjection In Live.
2. RYDTEngine
   Learning NSECM Risk From watch to NSEFO Risk Step by step how to do(Continue).

**Comment by Anil-Restalgo** [comment]
**13-10-2025**
=

1. RYDTEngine.
   Look Of Wrong Margin Rjection In Live Find Problem.

**Comment by Anil-Restalgo** [comment]
**14-10-2025 && 15-10-2025**
=

1. RYDTEngine.
   Look Of Wrong Margin Rjection In Live Find Problem.
   F_ATP And R_ATP Not Showing issue.
   CycleWiseATP Report Not Showing diff Showing only 0.

**Comment by Anil-Restalgo** [comment]
**16-10-2025**
=

1. RYDTEngine.
   TaskID945.

**Comment by Anil-Restalgo** [comment]
**17-10-2025**
=

1. RYDTEngine.
   TaskID945.

**Comment by Anil-Restalgo** [comment]
**28-10-2025**
=

1. RYDTEngine.
   TaskID945.

**Comment by Anil-Restalgo** [comment]
**29-10-2025**
=

1. RYDTEngine.
   TaskID945.

**Comment by Anil-Restalgo** [comment]
**30-10-2025**
=

1. RYDTEngine.
   TaskID945.
   Make The NSECM Common Utils Class For Add The Code of margin Calculation.
   Get The Key For Get Symbol Wise Var .
   Var File Load Done.
   Add ScriptMasterRow Class Add One Row and Of ISIN and Read ISIN From Contract File.

**Comment by Anil-Restalgo** [comment]
**31-10-2025**
=

1. RYDTEngine.
   TaskID945.
   Add The Calculation For NSECM Margin like how to do margin and how to get margin.
   Complete Pending Stock Margin but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)
   In Live :-
   Wrong Margine Rejection Check Log All Like Spinner Engine But not Get any Issue.

**Comment by Anil-Restalgo** [comment]
**01-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Add The Calculation For NSECM Margin like how to do margin and how to get margin.
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**03-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Testing In Colo Server of Margin Calculation but issue in pending sqoff margin like my buy qty is 20 and sell qty is 10 
   then we will get margin on 10 qty but margin show other this point is in Progress.
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**04-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Testing In Colo Server of Margin Calculation but issue in pending sqoff margin like my buy qty is 20 and sell qty is 10 
   then we will get margin on 10 qty but margin show other this point is in Progress.
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**05-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Testing In Colo Server of Margin Calculation but issue in pending sqoff margin like my buy qty is 20 and sell qty is 10 
   then we will get margin on 10 qty but margin show other this point is in Progress.
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**06-11-2025 && 07-11-2025**
=

1. RYDTEngine.
   Wrong Margin Issue IN Live(Share India).

**Comment by Anil-Restalgo** [comment]
**08-11-2025 && 09-11-2025 && 10-11-2025**
=

1. RYDTEngine.
   Wrong Margin Issue IN Live(Share India).

**Comment by Anil-Restalgo** [comment]
**11-11-2025 && 12-11-2025**
=

1. RYDTEngine.
   Wrong Margin Issue not get but in this exe.
   whenever all exe close at time net qty was 0 and margin also 0 but whenever exe open again at time net qty is 0 but margin show Issue.

**Comment by Anil-Restalgo** [comment]
**13-11-2025**
=

1. RYDTEngine.
   Wrong Margin Issue not get but in this exe.
   whenever all exe close at time net qty was 0 and margin also 0 but whenever exe open again at time net qty is 0 but margin show Issue Solved.
   Admin RMS Rejection issue in Live Work With VMD Done.

**Comment by Anil-Restalgo** [comment]
**17-11-2025 && 18-11-2025**
=

1. RYDTEngine.
   whenever all exe close at time net qty was 0 and margin also 0 but whenever exe open again at time net qty is 0 but margin show Get again in UAT Check.
   TaskID945.
   Add The Calculation For NSECM Margin like how to do margin and how to get margin.
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**19-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Add The Calculation For NSECM Margin like how to do margin and how to get margin.
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**20-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Get Margin on Refresh button (From My sideFrom My Side Check Pending By LokeshSir).
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**21-11-2025 && 22-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Get Margin on Refresh button.
   Download All File like BSEFO and NSEFO and NSECM.
   NSECM Margin Merge With NSEFO and BSEFO(Work In Process).
   Complete Pending Stock Margin and On Trade but Changes By Parameter pending.(From My Side Check Pending By LokeshSir)

**Comment by Anil-Restalgo** [comment]
**24-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Get Margin on Refresh button pending.
   Download All File like BSEFO and NSEFO and NSECM.
   NSECM Margin Merge With NSEFO and BSEFO.
   Testing With DeepakBhai Testing Done in My Branch not in UAT branch.
   All Complete by My Side Now Pending Code Discuss and Performance Level etc with Lokesh Sir.

**Comment by Anil-Restalgo** [comment]
**25-11-2025**
=

1. RYDTEngine.
   TaskID945.
   Get Margin on Refresh button pending.
   Download All File like BSEFO and NSEFO and NSECM.
   NSECM Margin Merge With NSEFO and BSEFO.
   Testing With DeepakBhai Testing Done in My Branch not in UAT branch.
   All Complete by My Side Now Pending Code Discuss and Performance Level etc with Lokesh Sir.
   Add Comment In All Changes and Add Place.
   Live Issue Check With Support Team Log and Etc. (Not Received Suare Off Finish Message)

**Comment by Anil-Restalgo** [comment]
**26-11-2025**
=

1. RYDTEngine.
   TaskID979.

**Comment by Anil-Restalgo** [comment]
**27-11-2025**
=

1. RYDTEngine.
   TaskID979.
   Done Don't Check Var in NSECM.
   Pending Don't Sort Selling.

**Comment by Anil-Restalgo** [comment]
**28-11-2025**
=

1. RYDTEngine.
   TaskID979.
   Done Don't Check Var in NSECM.
   Done Don't Sort Selling.
   TaskID945.
   Get Margin on Refresh button Done.

**Comment by Anil-Restalgo** [comment]
**29-11-2025**
=

1. RYDTEngine.
   TaskID190 and 251.
   R&D.

**Comment by Anil-Restalgo** [comment]
**02-12-2025**
=

1. RYDTEngine.
   TaskID183.
   R&D.

**Comment by Anil-Restalgo** [comment]
**03-12-2025**
=

1. RYDTEngine.
   TaskID183.
   R&D.
   TaskID986.
   In Engine, SQ process should be run if Session is in Open.
   Done Of TaskID986.

**Comment by Anil-Restalgo** [comment]
**04-12-2025**
=

1. RYDTEngine.
   TaskID183.
   R&D.
   In Live 
   Maargin Percent Get High and Margin Get High Issue When Engin Stop and Start Again.

**Comment by Anil-Restalgo** [comment]
**05-12-2025**
=

1. RYDTEngine.
   In Live 
   Maargin Percent Get High and Margin Get High Issue When Engin Stop and Start Again.

**Comment by Anil-Restalgo** [comment]
**08-12-2025 && 09-12-2025**
=

1. RYDTEngine.
   In Live 
   Maargin Percent Get High and Margin Get High Issue When Engin Stop and Start Again.
2. Live Issue.
   Show Live Issue With VMD and SupportTesting Team.
3. Self.
   Learning on MVC.

**Comment by Anil-Restalgo** [comment]
**10-12-2025**
=

1. Trading System:
	TaskID 991. In Normal Market Watch, Add Delta column and detla value logic same as rule add board.(Done)
	TaskID 992. In Normal Market Watch, Add Delta column and IV value.(Done)

**Comment by Anil-Restalgo** [comment]
**11-12-2025 && 12-12-2025**
=

1. Trading System:
   Understanding delta,IV formulas from VaibhavBhai and Excel Files and You tube Video.
   Check All formulas correct in our system as per vaibhavbhai and excel sheets.

**Comment by Anil-Restalgo** [comment]
**13-12-2025**
=

1. RYDTEngine.
   Change message of Session is close or preopen.
2. Trading System.
   R&D Form Last Location Save.

**Comment by Anil-Restalgo** [comment]
**15-12-2025**
=

1. Trading System.
   R&D Form Last Location Save.

**Comment by Anil-Restalgo** [comment]
**16-12-2025**
=

1. Trading System.
   NSECM Margin and TaskID 979 (Admin Module) and TaskID 945 Mearge in UAT and Testing By me and Some Changes Do Like Converted File Write and read.

**Comment by Anil-Restalgo** [comment]
**17-12-2025**
=

1. Trading System.
   Mearge TaskID 992,986,979,945 in UAT Changes In Comman Dll With VMD.
   R&D Form Last Location Save.

**Comment by Anil-Restalgo** [comment]
**18-12-2025 to 22-12-2025**
=

1. RYDT Risk Engine.
   TaskID 996 Risk Related Changes With Lokesh Sir And ViragBhai(Continue).

**Comment by Anil-Restalgo** [comment]
**23-12-2025 and 24-12-2025**
=

1. RYDT Risk Engine.
   TaskID 996 Risk Related Changes With Lokesh Sir And ViragBhai(Continue).

**Comment by Anil-Restalgo** [comment]
**30-12-2025 to 01-01-2026**
=

1. Trading System
   Testing With Deepak bhai and Pratik Bhai Margin.
   Watch with Virag bhai and Lokesh Sir DockPanel Forms Work.
2. RYDT Risk Engine.
   NSECM Margin Whenever get trade and do single sq off at time get margin of first sqoff trade.

**Comment by Anil-Restalgo** [comment]
**02-01-2026**
=

1. RYDT Risk Engine.
   NSECM Margin Whenever get trade and do single sq off at time get margin of first sqoff trade.

**Comment by Anil-Restalgo** [comment]
**03-01-2026 && 05-01-2026**
=

1. RYDT Risk Engine.
   NSECM Margin Whenever get trade and do single sq off at time get margin of first sqoff trade.

**Comment by Anil-Restalgo** [comment]
**06-01-2026**
=

1. RYDT Risk Engine.
   NSECM Margin Whenever get trade and do single sq off at time get margin of first sqoff trade Done and also Testing Done With DeepakBhai.

**Comment by Anil-Restalgo** [comment]
**07-01-2026**
=

1. RYDT Risk Engine.
   After Sqoff Process Finish on Last Trade Received then after right away get message of SqOff Process Finish.

**Comment by Anil-Restalgo** [comment]
**08-01-2026**
=

1. RYDT Risk Engine.
   After Sqoff Process Finish on Last Trade Received then after right away get message of SqOff Process Finish.
   After Restart Engine Get Margine 7 to 8 times more.

**Comment by Anil-Restalgo** [comment]
**09-01-2026**
=

1. RYDT Risk Engine.
   After Sqoff Process Finish on Last Trade Received then after right away get message of SqOff Process Finish Give For UAT Today.
   After Restart Engine Get Margine 7 to 8 times more For This Task What To do Tell us Lokesh Sir.

2. ProntoScheduler.
   BSEFO BhavCopy File Insert in SQL DataBase Server.

**Comment by Anil-Restalgo** [comment]
**10-01-2026**
=

1. ProntoScheduler.
   BSEFO BhavCopy File Insert in SQL DataBase Server(Done).

**Comment by Anil-Restalgo** [comment]
**21-01-2026 to 23-01-2026**
=

1. Transplexer.
   BSEFO All Files Download in and put in needed place.

**Comment by Anil-Restalgo** [comment]
**28-01-2026 to 30-01-2026**
=

1. Transplexer.
   BSEFO All Files Download in and put in needed place.
   in between My PC not Working Properly.
